golang/go-cloud-run-hello-world/README.md

```diff
@@ -11,6 +11,7 @@ This "Hello World" is a [Cloud Run](https://cloud.google.com/run/docs) service t
     2. [Running on Cloud Run](#intellij-running-on-cloud-run)
 * [Service Configuration](#service-configuration)
 * [Next steps](#next-steps)
+* [Contributing](#contributing)
```

Comment:
this doesn't seem to link anything?

(comment applies to all CR READMEs)

Reply:
Good catch. We moved contributing section to a separate file a while ago but this wasn't adjusted. Will remove.